Source/WebCore/dom/Attr.cpp

```diff
@@ -101,9 +102,31 @@ ExceptionOr<void> Attr::setPrefix(const AtomString& prefix)
 
 ExceptionOr<void> Attr::setValue(const AtomString& value)
 {
-    if (RefPtr element = m_element.get())
-        return element->setAttribute(qualifiedName(), value, true);
-    else
+    if (RefPtr element = m_element.get()) {
+        auto verifiedValue = value;
+        if (document().requiresTrustedTypes()) {
+            AttributeTypeAndSink attributeTypeAndSink = trustedTypeForAttribute(element->nodeName(),
+            qualifiedName().localName(), element->namespaceURI(), qualifiedName().namespaceURI());
+            if (!attributeTypeAndSink.attributeType.isNull()) {
+                auto attributeValue = trustedTypesCompliantAttributeValue(*document().scriptExecutionContext(),
+                    attributeTypeAndSink.attributeType, value, attributeTypeAndSink.sink);
+                if (attributeValue.hasException())
+                    return attributeValue.releaseException();
+
+                verifiedValue = AtomString { attributeValue.releaseReturnValue() };
+
+                if (!m_element.get()) {
+                    m_standaloneValue = verifiedValue;
+                    return { };
+                }
+
+                if (RefPtr newElement = m_element.get(); newElement != element)
+                    return { };
+            }
+        }
+
+        element->setAttribute(qualifiedName(), verifiedValue);
+    } else
         m_standaloneValue = value;
```

Comment:
Should do this to save a tiny bit of reference count churn:
```suggestion
        m_standaloneValue = WTFMove(value);
```

Reply:
Turns out this doesn't work on Linux ports because value is const.

Reply:
That makes sense, wouldn’t work on any port, not just Linux.

Reply:
Oddly it lets me build it on Apple platforms but the linux ones fail to build. Either way I've reverted it for here.